source/lib-intro.tex

```diff
@@ -3039,12 +3039,18 @@
 #define MB_CUR_MAX @\seebelow@
 
 namespace std {
+  // Exposition-only function type aliases
+  extern "C" using c_exit_t = void();                                // \expos
+  extern "C++" using cxx_exit_t = void();                            // \expos
```

Comment:
I removed `cxx_` and renamed the typedefs to `_fn`.

I like the placeholder style, but take note that we we will need to refer to that name in the itemdescls as well, which are quite far way (see updated PR).